Apps/Src/Temperature.c

```diff
@@ -141,15 +129,17 @@ ErrorStatus Temperature_ChannelConfig(uint8_t tempChannel) {
 		Minions[board].com.tx_data[5] = (0xF << 4) + AUX_I2C_NACK_STOP;
     }
 
-    // Send data
+	// Send data
     wakeup_sleep(NUM_MINIONS);
     LTC6811_wrcomm(NUM_MINIONS, Minions);
-    LTC6811_stcomm();
+	pec_error |= LTC6811_rdcomm(NUM_MINIONS, Minions);
```

Comment:
Don't we have a version of this function that automatically checks the PEC error and retries a few times if it fails? Can we use that?